### DISCUSSION TITLE
How to make all rpc ports accessible from outside？

### DISCUSSION THREAD

**Discussion by ygcool** [Chia-Network/chia-blockchain#5670] [Support]
Modify config.yaml which options allow all rpc to be accessed externally？

thanks

**Comment by BrandtH22** [comment]
Hey, since the rpcs allow for sending of funds it is not recommended nor will it be implemented in the chia reference client for rpcs to be accessed externally. 

Some third party developers have created ways to access the full node rpc remotely such as fireacademy: https://github.com/FireAcademy

Since this feature is not planned and this ticket is a number of years since a previous comment we will be closing the ticket but do not hesitate to reopen it if any questions arise.
